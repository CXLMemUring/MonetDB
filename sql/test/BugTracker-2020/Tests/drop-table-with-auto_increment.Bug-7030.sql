create schema s1;
create table s1.t (i int not null auto_increment);
drop table s1.t;
drop schema s1;
