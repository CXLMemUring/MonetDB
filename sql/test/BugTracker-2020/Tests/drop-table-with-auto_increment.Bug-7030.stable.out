stdout of test 'drop-table-with-auto_increment.Bug-7030` in directory 'sql/test/BugTracker-2020` itself:


# 12:21:18 >  
# 12:21:18 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-868236" "--port=30843"
# 12:21:18 >  

#create schema s1;
#create table s1.t (i int not null auto_increment);
#drop table s1.t;
#drop schema s1;

# 12:21:18 >  
# 12:21:18 >  "Done."
# 12:21:18 >  

